package.json

```diff
@@ -55,6 +57,7 @@
     "bs58": "^5.0.0",
     "buffer": "^6.0.3",
     "create-xpub": "^2.1.0",
+    "gh-pages": "^5.0.0",
```

Comment:
Note: `gh-pages` should be in `devDependencies` instead of `dependencies`.

Reply:
Ah, my bad. Will change that asap!